### PR TITLE
use system language by default

### PR DESCRIPTION
**Pull request by un1t**
Default language is english and settings menu is not available until you create first wallet. For not english speaking users it very hard to accomplish.

It would be better to use system language at first start.
My commit is applied to qt interface. I'm not sure where to set default language for kivy interface.

### CODE REVIEW

File: electrum/i18n.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 import gettext
 
+from PyQt5.QtCore import QLocale
```
Comment (SomberNight) [inline electrum/i18n.py]: you must not assume that qt is available, in the core lib.

Comment (un1t) [reply electrum/i18n.py]: You right. I also noticed this, after CI failed.

Comment (bauerj) [reply electrum/i18n.py]: Can you move the function to e.g. `electrum.gui.qt.util` in stead?

File: electrum/i18n.py [hunk 2]
```diff
@@ -41,6 +43,11 @@ def set_language(x):
         language = gettext.translation('electrum', LOCALE_DIR, fallback=True, languages=[x])
 
 
+def get_default_language():
+    system_locale = QLocale.system().name()
+    return languages.get(system_locale, 'en_UK')
```
Comment (bauerj) [inline electrum/i18n.py]: Wouldn't this return something like `"German"` if `system_locale` was `de_DE`?

I think something like 

```python
return system_locale if system_locale in languages else "en_UK"
```

would make more sense.

Comment (un1t) [reply electrum/i18n.py]: Oops, yeah, you right!

Comment (un1t) [reply electrum/i18n.py]: I made this fix and moved this function to gui.qt.utils.

**Pull request merged**